app/views/shared/_stats.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="crayons-tabs crayons-tabs--analytics">
-  <button class="crayons-tabs__item crayons-tabs__item--current" id="week-button">Week</button>
-  <button class="crayons-tabs__item" id="month-button">Month</button>
-  <button class="crayons-tabs__item" id="infinity-button">Infinity</button>
+  <button data-text="Week" class="crayons-tabs__item crayons-tabs__item--current" id="week-button">Week</button>
```

Comment:
(blocking) - again I think it must be the css changes, but these tabs used to live in the middle of the page:

![screenshot of analytics tabs right-aligned on the page](https://user-images.githubusercontent.com/20773163/117010156-6c094580-ace4-11eb-8ddb-0d01b59a1c75.png)

Also looks like the text alignment has gone wrong in mobile:

![screenshot of tabs with text pushed up to the far left instead of centered](https://user-images.githubusercontent.com/20773163/117010273-8d6a3180-ace4-11eb-8a0d-9d6aadbfb01d.png)

Reply:
First one was a change that was made after I started the PR (when I moved it to tabs I placed them on the right but they were moved to the middle in a later PR) I've put it in the middle now.

And the second one was caused by using flex rather than block leading to text-align not working. also fixed.